trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core.tests/src/org/eclipse/tracecompass/incubator/trace/server/jersey/rest/core/tests/stubs/XySeriesStub.java

```diff
@@ -56,14 +64,21 @@ public class XySeriesStub implements Serializable {
     @JsonCreator
     public XySeriesStub(@JsonProperty("seriesName") String name,
             @JsonProperty("seriesId") Integer id,
-            @JsonProperty("xValues") ISamplingStub xValues,
+            @JsonProperty("xValues") long[] xValues,
+            @JsonProperty("xCategories") List<String> categories,
+            @JsonProperty("xRanges") List<RangeStub> ranges,
```

Comment:
Oops, I was looking at JsonProperty names for the comments above regarding Javadoc. So it's ok with you want to use x-prefix or not.